src/Elastic.Markdown/Assets/toc-nav.ts

```diff
@@ -137,4 +137,12 @@ export function initTocNav() {
 	window.addEventListener('scroll', update);
 	window.addEventListener('resize', update);
 	setupSmoothScrolling(elements);
+	setupReportIssue();
+}
+
+function setupReportIssue() {
+	var currentPage = window.location.href;
+	var reportIssue = document.getElementById('report-issue-link');
+	if (reportIssue)
+		reportIssue.setAttribute('href', `https://github.com/elastic/docs-content/issues/new?template=issue-report.yaml&link=${currentPage}`);
```

Comment:
Per https://github.com/elastic/docs-content/pull/886#discussion_r2008043424, can you please update this URL to also include adding a `source:web` label to issues. I believe the query string is `&labels=source%3Aweb`